### PR TITLE
Job Schedule concurrency policy

### PR DESCRIPTION
**Pull request by taxone**
https://github.com/spring-cloud/spring-cloud-deployer-kubernetes/issues/494

### CODE REVIEW

**Review by onobc — CHANGES_REQUESTED** [review]
Hi @taxone

Thank you for the contribution. I apologize for the wait on the review.

File: src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesScheduler.java [hunk 1]
```diff
@@ -202,6 +204,19 @@ protected CronJob createCronJob(ScheduleRequest scheduleRequest) {
 				schedulerProperties.get("spring.cloud.deployer.kubernetes.cron.expression") :
 				schedulerProperties.get(SchedulerPropertyKeys.CRON_EXPRESSION);
 		Assert.hasText(schedule, "The property spring.cloud.deployer.cron.expression must be defined");
+		
+		String concurrencyPolicy = schedulerProperties.get(KUBERNETES_DEPLOYER_CRON_CONCURRENCY_POLICY);
+		// check default server properties
+		if (!StringUtils.hasText(concurrencyPolicy)) {
+			concurrencyPolicy = this.properties.getCron().getConcurrencyPolicy();
```
[the remark this answers is not in the document]
Comment (taxone) [reply src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesScheduler.java:211]: No, it doesn't. 
The intent of that code is using the default server configuration if the value is not provided for the current schedule.
This is the default server configuration:

image

The schedule is created without providing any concurrency policy in the deployment properties.
You can see a debugging session below:

image


The *scheduleProperties* map does not contain the concurrency policy. But it's present in the **KubernetesDeploymentProperties**:

image

Do you agree?

[the remark this answers is not in the document]
Comment (onobc) [reply src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesScheduler.java:211]: Hi @taxone - got really busy over here and just picking this back up. 

Nice touch on the debugger screenshots. And yes, I do agree w/ you. I see where the scheduler props are merged w/ the current schedule props.

**Review by onobc — APPROVED** [review]
Nice work @taxone . Thanks again for the contribution and I apologize it took me a bit to get back to the review. Merging now.

**Pull request merged**